frontend/src/modules/details/view.jsx

```diff
@@ -32,6 +32,7 @@ import logoNotFound from "../../images/logo-not-found.png";
 import uniqBy from "lodash/uniqBy";
 
 const currencyFormat = (curr) => Intl.NumberFormat().format(curr);
+const urlLink = (url) => (url.indexOf("http") !== 0 ? `https://${url}` : url);
```

Comment:
I think it is reasonable to assume that we want to prefix `https`, since the form shows `https` as the prefix. But, I wonder if it may lead to problems in rare cases.

Reply:
The other option is to prefix `http` and assume the link has redirect in place